### ISSUE TITLE
Where are builds published?

### ISSUE THREAD

**Issue by csandersdev** [cqframework/cql-translation-service#20]
I want to include link to the JAR of the this service in a client library I am building so that I can take advantage of the constants (PARAM_TO_OPTIONS_MAP, ELM_JSON_TYPE, ELM_XML_TYPE, etc.) and utilize the server for testing purposes. I looked in both the [Maven Central](https://repo1.maven.org/maven2/org/mitre/) and Sontatype Public repositories and the jar doesn't exist in either location. Are the builds somewhere else? I understand that it gets published as a Docker image, but I want the Java dependency.

This is what is failing to resolve:
```
		<dependency>
			<groupId>org.mitre.bonnie</groupId>
			<artifactId>cqlTranslationServer</artifactId>
			<version>1.4.8</version>
		</dependency>
```

**Comment by cmoesel** [comment]
No, I don't believe we've published this as a library to Maven Central or Sonatype.  So far, I think most users of this service have either used docker or built the jar locally.  I don't know if `org.mitre.bonnie` is the right group ID to use at this point.  We might want to consider using the same group as the cql-to-elm libraries.  @brynrhodes -- what do you think?

**Comment by brynrhodes** [comment]
@cmoesel , @csandersdev , yes, happy to add support for publishing these to the cqframework.info, that makes sense since the repository is in the same github organization.

**Comment by msargentibm** [comment]
Hi folks -- was there any movement on publishing this as a library in cqframework.info?
